### PR TITLE
New package: grandorgue-3.13.0

### PR DESCRIPTION
**Pull request by nicolodavis**
#### Testing the changes
- I tested the changes in this PR: YES

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): YES

#### Local build testing
- I built this PR locally for my native architecture, (x86_64 glibc)

### CODE REVIEW

**Comment by nicolodavis** [comment]
Just curious if I need to do anything at my end to get the CI to green tick this PR?

**Comment by nicolodavis** [comment]
Hi @classabbyamp, is Void still accepting new package contributions? Let me know if I should close this PR instead.

**Comment by classabbyamp** [comment]
we do accept new package contributions, but generally it requires a maintainer to be either interested in the package or see the utility of merging it

we also generally like to see new contributors make some contributions (updates, fixes, etc) to existing packages (especially orphaned packages) before adding new packages.

File: srcpkgs/grandorgue/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'grandorgue'
+pkgname=grandorgue
+version=3.13.0
+revision=1
+archs="x86_64"
+build_style="cmake"
+configure_args="-DCMAKE_BUILD_TYPE=Release -G Ninja"
```
Comment (classabbyamp) [inline srcpkgs/grandorgue/template:7]: neither of these need to be set. void sets the build type to None and the generator to ninja by default in `common/build-style/cmake.sh`

File: srcpkgs/grandorgue/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+depends="gettext po4a zip"
+short_desc="Pipe Organ Simulator"
+maintainer="Nicolo Davis <nicolodavis@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/GrandOrgue/grandorgue"
+
+export WX_CONFIG=wx-config-gtk3
+
+do_fetch() {
+	git clone --recurse-submodules --depth 1 --branch ${version}-${revision} https://github.com/GrandOrgue/grandorgue.git ${wrksrc}
```
Comment (classabbyamp) [inline srcpkgs/grandorgue/template:19]: do not use `git clone` to fetch. if a packages has submodules, they should be added as distfiles. see the `edk2-ovmf` template for an example.

File: srcpkgs/grandorgue/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+configure_args="-DCMAKE_BUILD_TYPE=Release -G Ninja"
+hostmakedepends="pkg-config git ImageMagick"
+makedepends="jack-devel eudev-libudev-devel zlib-devel yaml-cpp-devel wxWidgets-devel wxWidgets-common-devel wxWidgets-gtk3-devel fftw-devel wavpack-devel alsa-lib-devel"
+depends="gettext po4a zip"
+short_desc="Pipe Organ Simulator"
+maintainer="Nicolo Davis <nicolodavis@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/GrandOrgue/grandorgue"
+
+export WX_CONFIG=wx-config-gtk3
```
Comment (classabbyamp) [inline srcpkgs/grandorgue/template:16]: this can probably be replaced with `build_helper="cmake-wxWidgets-gtk3"`

**Pull request closed without merging**